Assets/FantasyEnvironments/Environments/Scenes/DemoScene.unity

```diff
@@ -38,7 +38,7 @@ RenderSettings:
   m_ReflectionIntensity: 1
   m_CustomReflection: {fileID: 0}
   m_Sun: {fileID: 2003931976}
-  m_IndirectSpecularColor: {r: 0.4465782, g: 0.49641252, b: 0.5748167, a: 1}
+  m_IndirectSpecularColor: {r: 0.44657838, g: 0.49641234, b: 0.57481676, a: 1}
```

Comment:
just a reminder to try to discard these changes if you can, otherwise whoops

Reply:
wait this is in fantasyEnvironments not the normal DemoScene so honestly you're fine

Reply:
might've been an accident? I will try to remember